cognite/client/_api/data_modeling/instances.py

```diff
@@ -623,6 +623,13 @@ def apply(
                 ...     auto_create_end_nodes=True
                 ... )
 
+            Using helper function to create valid graphql timestamp for a datetime object:
+
+                >>> from cognite.client.utils import datetime_to_ms_iso_timestamp
+                >>> from datetime import datetime
+                >>> import pytz
+                >>> my_date = datetime(2020, 3, 14, 15, 9, 26, 535000, tzinfo=pytz.utc)
```

Comment:
This felt much better! I just think we should change `pytz` to built-in:
```suggestion
                >>> from datetime import datetime, timezone
                >>> my_date = datetime(2020, 3, 14, 15, 9, 26, 535000, tzinfo=timezone.utc)
```